README.md

```diff
@@ -132,7 +136,9 @@ far, `Method_1` and `Method_2` returned `'Empty` bodies, but we can fix that.
 import           Data.Text (Text)
 import qualified Serv.ContentType as Ct
 
-type Method_3 = 'Method 'GET '[] ('Body '[ Ct.TextPlain ] Text)
+type RawBody = HasBody '[ Ct.TextPlain ] Text
```

Comment:
Sounds good to me. Let me know when something feels like a good draft then I'm happy to once-over it.